### ISSUE TITLE
YouTube link is broken

### ISSUE THREAD

**Issue by skyiea** [forbesmyester/SyncIt#11]

**Comment by forbesmyester** [comment]
You're right, I never owned that video, it was from LNUG and I just linked to it... Will have a word with them and find out what's going on.

Thanks

**Comment by forbesmyester** [comment]
Disappointingly it looks like the video is lost. The account that uploaded
it (which was not mine) has been deleted and it took all the videos with it
:-(



On 25 Jul 2017 8:42 am, "Serhii Muryhin" <notifications@github.com> wrote:

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/forbesmyester/SyncIt/issues/11>, or mute the thread
